django-verdant/rcasite/settings/base.py

```diff
@@ -92,6 +92,37 @@
 #    'django.contrib.staticfiles.finders.DefaultStorageFinder',
     'compressor.finders.CompressorFinder',
 )
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```

Comment:
Not even sure if `STATICFILES_STORAGE` is used for v1.6.3

Reply:
Should be, but Whitenoise is only tested on >Django 1.8 - so there may be other issues here. If it works locally it should work in production though.